### ISSUE TITLE
FEATURE - SWISH itself becomes a CPACK ?  This becomes cpack_swish_examples?

### ISSUE THREAD

**Issue by TeamSPoon** [ClioPatria/swish#1]
I sort of assume this project exists for several reasons:

SWISH administrators really *want* and *need* the HTTP Admin tools of ClioPatria 

   http://cliopatria.swi-prolog.org/admin/settings
   http://cliopatria.swi-prolog.org/admin/configuration
   http://cliopatria.swi-prolog.org/user/statistics
   http://cliopatria.swi-prolog.org/help/http
   http://cliopatria.swi-prolog.org/cpack/list_packages

Running SWISH and ClioPatria together makes a system that is *far* greater than the sum of those two parts.    So myself, I *LOVE* this project's intent.    

Despite so far not happening,  this project serves to mitigate any incompatibility created by making SWISH a CPACK doesn't damage the actual SWISH Project.  

We need a home for examples *too* "ClioPatria specific" to be in SWISH source management.  

Anyway, I wanted to just point out what I think I knew and wanted to get further insight if there was more reasoning beyond the two above last reasons for not making the actual SWISH Project itself a CPACK (Since as far as I can tell it merely add the "applications" and "rdf" directories ?

**Comment by JanWielemaker** [comment]
I agree the sum of these is great.  SWISH __is__ a cpack though, so I don't really see the feature request?  What do I miss?  Note that you do need the latest Prolog and ClioPatria to install the latest version of the `swish` cpack.

**Comment by TeamSPoon** [comment]
I couldn't yet tell if leaving out parts to Standalone SWISH was wanted or not.  But enough other changes are made examples/index.json, page.pl to make it more ClioPatria Based left out parts may been wanted.   This project is microcosmic example of "extending" of SWISH at the same time being a CPACK.  

------------------------------------------------------

ClioPatria upgrades the RDF renderer  (++GOOD) 

Profiles - starter Notebooks/Prolog  seem to only be enumerated out of only one profile directory.  Thus StandAlone SWISH's notebooks are excluded.    (Shouldn't they be aggregated?)

Examples - are enumerated from all example directories this findable (++GOOD)

The 'page.pl's are different. ( This was wanted - this is ClioPatria SWISH )

But the Examples JSON is one or the other.  Like page.pl right?   But if those examples are missing from the menu I will **also** need to  install StandAlone SWISH.  Pretend I do not want to do that.

----------------------------------------------------

So what I am really *feature-wise* after is it would be nice if it extended StandAlone SWISH (@SWI-Prolog/swish)  without removing parts of standalone swish.

To and have it inserting the ClioPatria features without needing to replace files in StandAlone SWISH since my version of StandAlone SWISH might already be different fork of the the one in SWI-Prolog/swish.

So I thought about how one accomplishes this goes like...
1) Make StandAlone SWISH installable as CPACK without any ClioPatria features
2) Make ClioPatria features installable into SWISH without removing standalone features

This way I can use my forks of #1 and can still use it wis ClioPatria features #2 


Ps
In an ideal world though no one forks SWISH.  They make Extensions for SWISH #2s 
 For me: "Make PrologMUD specific features available in SWISH without forking it  "
   (For instance i intercept http calls to index.json and reply with my new additional materials. oops, only works once)

**Comment by TeamSPoon** [comment]
p.p.s.

In an ideal world though no one needs to fork StandAlone SWISH.  They make Extensions for SWISH.  And if they are going that far (customizing SWISH) it might make sense that these ways people (myself and others) will probably want to manage these SWISH customizations as CPACKS (under ClioPatria). 

( I know, really most customizations of SWISH could/should be done if at all possible by only "adding Notebooks"  ... and one day that might eventually be possible ...)
